### ISSUE TITLE
Generated mermaid files do not implement the --module-names option of pyreverse.

### ISSUE THREAD

**Issue by frank-pet** [pylint-dev/pylint#10164]
### Bug description

```python
The mermaid files generated by the -ommd option do not implement  the -m, --module-names option.  An inspection of pyreverse/mermaidjs_printer.py shows that the module information indeed present and removed before outputting it to the mermaid file.

Note that -m[y|n] works fine for .dot files...
```

### Configuration

```ini
pylint/pyreverse/mermaidjs_printer.py
```

### Command used

```shell
pyreverse -A -my -f ALL -ommd filename.py
```

### Pylint output

```python
valid mermaid file with only the classnames in the output.
```

### Expected behavior

Would expect to see "modulename.classname" in the mermaid file, vs just "classname".

### Pylint version

```shell
pylint 2.16.2, but also in 3.3.0 at lines 60, 76, and 77.
```

### OS / Environment

debian bookworm, arm64

### Additional dependencies

```python
none for this bug
```

**Comment by jbutz** [comment]
It's not a long-term fix, and hasn't been tested outside my own use case, but if you search through the dependencies for your code and can find the `mermaidjs_printer.py` file for pyreverse add a `#` (comment) in front of the following lines:

- `name = name.split(".")[-1]` in the `emit_node` method
- `from_node = from_node.split(".")[-1]` and `to_node = to_node.split(".")[-1]` in the `emit_edge` method
